processors/src/main/java/com/axellience/vuegwt/processors/component/template/parser/TemplateParser.java

```diff
@@ -122,6 +133,27 @@ private void processImports(Source doc)
             .forEach(outputDocument::remove);
     }
 
+    private String processScopedCss(Source doc) {
+        class Rslt { String scopedCss = ""; }
+        final Rslt rslt = new Rslt();
+        doc.getAllElements().stream()
+            .filter(element -> "style".equalsIgnoreCase(element.getName())
```

Comment:
Maybe add a private named method for that predicate to make it easier to read? Like:
```java
private boolean isScopedStyleElement(Element)
```

And then:
```java
.filter(this::isScopedStyleElement)
```